### DISCUSSION TITLE
Welcome to the Deep Learning Fundamentals discussion forum!

### DISCUSSION THREAD

**Discussion by rasbt** [Lightning-AI/dl-fundamentals#1] [Lecture videos and quizzes]
Hey there 👋  

Welcome to the *Deep Learning Fundamentals* discussion forum!

Found any of the lectures confusing or have follow-up questions? 
Please don't hesitate to ask!

**Comment by oliverlibaw** [comment]
Really enjoying the class so far. Thank you for creating it! 

2 quick thoughts/requests: 

* I have no professional coding experience (just some self-taught python fundamentals) and I've been having some trouble with some basic github setup stuff. Would love some suggestions for YT videos or other resources there. Or perhaps if this forum gets pretty active, there could be a section for (sometimes remedial) technical troubleshooting. 

* The info in the course becomes so much more concrete when I'm fiddling with actual code. I'd love some more bite-sized problems I could fiddle with, in addition to the code examples and quiz questions. 

tx!

**Reply by rasbt** [reply]
Thanks for the super nice feedback @oliverlibaw! 

We actually made some intro videos for GitHub earlier this year that might be helpful. Have a look at episodes 6 - 9 [here](https://lightning.ai/pages/category/education/lightning-bits/)

> The info in the course becomes so much more concrete when I'm fiddling with actual code. I'd love some more bite-sized problems I could fiddle with, in addition to the code examples and quiz questions.

Hah, I may actually also have some answer for that part. Have you tried the exercises for the respective units? I must admit they might be a bit easy to miss ...

<img width="1064" alt="Screenshot 2022-12-30 at 5 54 39 PM" src="https://user-images.githubusercontent.com/5618407/210118799-f6d90388-ace6-4525-a20f-61e896936b1c.png">

**Reply by oliverlibaw** [reply]
Excellent! I hadn't gotten that far yet : )

**Comment by oliverlibaw** [comment]
<img width="810" alt="Screenshot 2023-01-02 at 8 27 35 PM" src="https://user-images.githubusercontent.com/92615450/210300274-08383dcf-9889-4f21-9d5c-26716ff40145.png">
bit of a typo here btw  : )

**Reply by rasbt** [reply]
Oooops, thanks! Yes, for sure, it should be `out_features=1` for sure! Will get that fixed!

**Comment by naomilago** [comment]
> Hey there 👋
> 
> Welcome to the _Deep Learning Fundamentals_ discussion forum!
> 
> Found any of the lectures confusing or have follow-up questions? Please don't hesitate to ask!

Hi there, folks!
I'm really happy to join this course. It seems pretty well conducted and I like the exercises. 
I hope to improve my knowledge on Deep Learning for NLP ✨

**Reply by rasbt** [reply]
Awesome! I am always happy to hear that the course is useful!! And please don't hesitate to reach out in case you have any questions!

**Comment by adoodle04** [comment]
Hello everyone, really glad to be here! And did you know, TensorFlow is the fourth most used between the data analysis and machine learning technologies, with 11.5% of professional developers using it, according to Stack Overflow?

**Reply by rasbt** [reply]
Thanks for sharing! Do you know the other "top 3" by chance?

**Comment by Mohamed-Rayane-LAKEHAL** [comment]
Good evening members, 

I was reading the paper of the GhostNet CNN model, and there is something i did not understand, author said that the first ghost block in the ghost bottleneck is for channels expansion, while the ghost module consists of a point-wise conv followed by a depth-wise one, for expansion it sounds correct, but they said the second ghost module is for channels reduction !, how can a point-wise followed by a D-wise be used for reduction... ? thank you.
